high_card.rb

```diff
@@ -1,5 +1,52 @@
+require "pry"
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
+@ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+@suits = [ "hearts", "spades", "clubs", "diamonds" ]
```

Comment:
There's no need to define these as instance variables if they exist outside of a class.